test/visibility/import/enablesUnqualified/multipleUnqualified/operatorsFirst.chpl

```diff
@@ -0,0 +1,558 @@
+module DefinesOp {
```

Comment:
Is this different from other `module DefinesOp` test code? Could we have just one copy of it and symbolically link or have COMPOPTS include it in the module search path?

Reply:
Eh, I think there's value in testing that it works both living inside and outside the file that uses it.  It's probably worth having both strategies for each style of use/import involved, but that also seemed like going a bit too deep on something that so far as I can tell does currently work.

Reply:
Would submodule-in-separate-file allow you to have this cake and eat it too?

Reply:
I think of that more as a third way that would warrant testing on its own